app/src/main/res/layout/item_weather.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout

```

Comment:
WARNING: [UnusedResources] The resource `R.layout.item_weather` appears to be unused
 Unused resources make applications larger and slow down builds.